frontend/src/components/DialogNewFeatures.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Button, Dialog, DialogActions, DialogTitle, ListItem, ListItemIcon, ListItemText, makeStyles } from '@material-ui/core'
+import { fontSize } from '@remote.it/components/lib/styles/variables'
+import { Icon } from '../components/Icon'
+
+type Props = { open: boolean; onClose: () => void }
+
+export const DialogNewFeatures: React.FC<Props> = ({ open, onClose }) => {
+
+  const css = useStyles({})
+  return (
+    <div>
```

Comment:
You can use a Route container here 
```suggestion
    <Route path="/devices/welcome">
```